examples/bench-http-request/source/app.d

```diff
@@ -93,9 +93,9 @@ void benchmark()
 void main()
 {
 	import vibe.core.args;
-	readOption("c", &g_concurrency, "The maximum number of concurrent requests");
-	readOption("d", &g_requestDelay, "Artificial request delay in milliseconds");
-	readOption("k", &g_maxKeepAliveRequests, "Maximum number of keep-alive requests for each connection");
+	readOption("c", cast(long*) &g_concurrency, "The maximum number of concurrent requests");
+	readOption("d", cast(long*) &g_requestDelay, "Artificial request delay in milliseconds");
+	readOption("k", cast(long*) &g_maxKeepAliveRequests, "Maximum number of keep-alive requests for each connection");
```

Comment:
A better design would be to make those variables `immutable` and to set them from a shared static constructor.

Reply:
but that would require duplicating these three variables, wouldn't it?
(then I could use `atomicStore` as well)